cmake/msys/msys_helper.sh

```diff
@@ -90,33 +90,33 @@ if [ $? -ne 0 ]; then
 	if [ $? -ne 0 ]; then
         	err "ERROR: Could not build/install fltk -- Zyn needs this.  Exiting."
 	fi
-	
-#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe 	
+
+#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe
 	popd
 else
-	warn "  - Skipping, fluid binary already exists" 
+	warn "  - Skipping, fluid binary already exists"
 fi
 
 info "Downloading and building libogg $oggver"
 
-if [ ! -e $mingw_root/lib/libogg.dll.a ]; then 
-	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O $HOME/libogg-source.tar.xz
+if [ ! -e "$mingw_root/lib/libogg.dll.a" ]; then
+	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O "$HOME/libogg-source.tar.xz"
```

Comment:
> Because people will continue to copy and paste it (as they have previously), and we may calculate the values.

Indeed...

> And $HOME is? I've used a lot of systems and I can promise you %20 in a URL is more common than <space> in a /home path.

Space in an url yes, but this is a version number, in a file name in an url... who's crazy enough to put a space here ? :p We're not setting `$HOME` here, so...

> @zapashcanon arguably... all this PR does is make the code harder to read, that portion is quite irrelevant.

I agree other parts are making the code harder to read, but, I think it's not a reason to avoid doing it when it's possible. :)

> We should add it separately. We only have a handful of scripts, and this is one more greenlight we can have for the PR process.

Actually, I though I'd add a separate build for things like this, with a first PR for shellcheck (just a matter of adding a shell script and a few lines to travis config file), then, doing a second PR to add astyle and kwstyle in this build as it will recquire more changes. But, I'll talk of this somewhere else.

> That said, it's fine to merge without. I won't hold up the patch for this alone.

Yep, I think (and would prefer if) the PR can be merged like this too.